src/property-filter/property-filter-autosuggest.tsx

```diff
@@ -164,6 +164,8 @@ const PropertyFilterAutosuggest = React.forwardRef(
       isEmpty,
       onRecoveryClick: handleRecoveryClick,
       hasRecoveryCallback: !!onLoadItems,
+      getErrorIconAriaLabel: () => undefined,
+      getRecoveryText: () => props.recoveryText,
```

Comment:
There is no i18n support for these strings in property filter. I will create a follow up ticket for that